test/testE2E.t.sol

```diff
@@ -53,11 +57,16 @@ contract E2ETests is Test {
 
     StubXcmUtils private constant xcmUtils = StubXcmUtils(XCM_UTILS_ADDRESS);
 
+    XcmTransactorV2.Multilocation public fakeFeeLocation;
+
     function setUp() public {
         token = new TestToken(1_000_000 * 10 ** 18);
         registry = new ParachainRegistry();
         staking = new ParachainStaking(address(registry), address(token));
         gov = new ParachainGovernance(address(registry), fakeTeamMultiSig);
+        parachain = new TestParachain(address(registry));
+        // setting feeLocation as native token of destination chain
+        fakeFeeLocation = XcmTransactorV2.Multilocation(0, parachain.x1External(3));
```

Comment:
I think this multilocation is incorrect. It should be pointing to an asset on the destination consumer parachain from the context of the evm parachain. See https://github.com/tellor-io/tellor-pallet/blob/f5bf011eca27cb19015c52d57d3f60734b7bc8a3/src/xcm/mod.rs#L227 for more info.

It should probably be something like https://github.com/tellor-io/tellor-pallet/blob/f5bf011eca27cb19015c52d57d3f60734b7bc8a3/src/xcm/mod.rs#L232